From c552c1c053e5f3ed42c45fa04aeb554ba6d51a61 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 16 Apr 2020 22:00:01 +0300
Subject: [PATCH] [SW-11770] hl: goya2_arc: install: add boot fit

In version 0.8.0, we also provide a FIT file which can be used to
program the SPI flash.
Later (0.9.0+), we will remove regular SPI flash binaries image.

Change-Id: Icad9f4d32016c1ded10b57e7042f50c1b3c7268e
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/scripts/make/goya2_arc/Makefile b/scripts/make/goya2_arc/Makefile
index a5d8e62..f1e3f18 100644
--- a/scripts/make/goya2_arc/Makefile
+++ b/scripts/make/goya2_arc/Makefile
@@ -31,6 +31,7 @@
 	sshpass -p123456 $(SCP) $(PPBOOT_BASE)/preboot	root@$(HL_INSTALL_ONIX_LAB_IP):/lib/firmware/habanalabs/$(HL_DEV)/
 	sshpass -p123456 $(SCP) $(PPBOOT_BASE)/ppboot	root@$(HL_INSTALL_ONIX_LAB_IP):/lib/firmware/habanalabs/$(HL_DEV)/
 	sshpass -p123456 $(SCP) $(IMAGES_DIR)/$(HL_DEV)-spi-flash.bin	root@$(HL_INSTALL_ONIX_LAB_IP):/lib/firmware/habanalabs/$(HL_DEV)/
+	sshpass -p123456 $(SCP) $(IMAGES_DIR)/$(HL_DEV)-boot-fit.itb	root@$(HL_INSTALL_ONIX_LAB_IP):/lib/firmware/habanalabs/$(HL_DEV)/
 
 .PHONY: goya2_arc-pldm-install
 goya2_arc-pldm-install: goya2_arc-pldm-target
